package.json

```diff
@@ -22,7 +22,11 @@
   "dependencies": {
     "@hapi/hapi": "20.2.2",
     "applicationinsights": "2.7.0",
-    "ffc-messaging": "2.6.5"
+    "ffc-messaging": "2.6.5",
+    "pg": "8.7.3",
+    "pg-hstore": "2.3.4",
+    "sequelize": "^6.32.0",
```

Comment:
No need for `^`

Reply:
Not sure why that's there, but will remove thanks